src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1395,8 +1395,8 @@ function _vcat(dfs::AbstractVector{<:AbstractDataFrame};
         end
     elseif cols === :setequal || cols === :equal
         if cols === :equal
-            Base.depwarn("`cols=:equal` is deprecated." *
-                         "Use `:setequal` instead.", :vcat)
+            throw(ArgumentError("`cols=:equal` is not supported. " *
```

Comment:
Below you removed this check in similar cases. Better be consistent.

Reply:
I removed them below, because below they were dead code - the warning there was impossible to be reached (it was a leftover of many changes that was forgotten to be removed earlier). Here the warning could be printed, so I left to follow the rule to try turning warnings into informative errors.